filters/auth/oidc.go

```diff
@@ -234,7 +235,7 @@ func (s *tokenOidcSpec) CreateFilter(args []interface{}) (filters.Filter, error)
 		return nil, err
 	}
 
-	subdomainsToRemove := 1
+	subdomainsToRemove := s.options.SubdomainsToRemove // default from config
```

Comment:
Hm, this will break for those who configure auth.OidcOptions manually - now it becomes zero by default when not specified. Here https://github.com/zalando/skipper/pull/2203#discussion_r1073836873we decided to go with a pointer.

Reply:
Thanks for the hint, good point!

Reply:
In general I think we have a lot of overlapping functionality between OIDC and OAuth Grant Flow related to cookie handling so I created https://github.com/zalando/skipper/issues/3455